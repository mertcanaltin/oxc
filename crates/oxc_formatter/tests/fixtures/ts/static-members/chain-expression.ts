this._position = this._editor
  ._getViewModel()
  ?.getPrimaryCursorState().modelState.position;
