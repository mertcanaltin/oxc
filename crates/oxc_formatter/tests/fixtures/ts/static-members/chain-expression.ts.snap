---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
this._position = this._editor
  ._getViewModel()
  ?.getPrimaryCursorState().modelState.position;

==================== Output ====================
this._position = this._editor
  ._getViewModel()
  ?.getPrimaryCursorState().modelState.position;

===================== End =====================
